src/test/java/org/folio/oaipmh/helpers/records/RecordMetadataManagerTest.java

```diff
@@ -76,7 +77,7 @@ class RecordMetadataManagerTest {
   private RecordMetadataManager metadataManager = RecordMetadataManager.getInstance();
   private StorageHelper storageHelper = new SourceRecordStorageHelper();
 
-  @Test
+  //@Test
```

Comment:
Missed one. Can we see the failures and determine is simple change to assertion could fix? Or is this issue timeboxed?